### ISSUE TITLE
corpus name needs to be unique across different users

### ISSUE THREAD

**Issue by carinedengler** [hipster-philology/pyrrha#190]
**Describe the bug**

adding a new corpus with the name `new` as user _foo_ is not possible if user _bar_ already has a corpus with such a name, leading to the following error message

`You have already a corpus going by the name corpus `

even if the user foo does not have any corpora associated with them at all

**To Reproduce**

1. add user _foo_
2. add user _bar_
3. add corpus with name  `new` as user _bar_
4. try to add corpus with name  `new` as user _foo_

**Expected behavior**
a user should be able to add a corpus with a name which is already in use by a corpus of another user

**Comment by PonteIneptique** [comment]
You're totally right, we should probably add the creator name in a secondary field, or it's ID (which would make sense to be honnest)

However, as you can share corpora, you could find yourself in a situation where a user have access to identically named corpora. So on the front, we should recoved and show this when a user have two corpora with the same name (not always, that would be clunky ?)
